src/sbvr-api/uri-parser.coffee

```diff
@@ -83,11 +100,11 @@ parseODataChangeset = (env, b) ->
 
 	if b.url[0] == '/'
 		{ url, apiRoot } = splitApiRoot(b.url)
-		odata = odataParser.matchAll(url, 'Process')
+		odata = memoizedParseOdata(url, 'Process')
```

Comment:
Do we need the `'Process'` argument here? Not sure where this ends up.

Reply:
Yep, that's a leftover from before - removed